authorizer-app-backend/src/main/java/org/radarbase/authorizer/api/ApiDeclarations.kt

```diff
@@ -41,6 +41,12 @@ data class RestOauth1UserId(
     @JsonProperty("userId") var userId: String,
 )
 
+data class SignRequestParams(
+    var url: String,
+    var method: String,
+    val params: MutableMap<String, String?>
```

Comment:
It seems that a `MutableMap` is not needed, a `Map` would suffice. This logic extends to places where this class is used.